Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -2839,6 +2839,8 @@ void RenderLayer::paintSVGResourceLayer(GraphicsContext& context, GraphicsContex
     auto localPaintDirtyRect = LayoutRect::infiniteRect();
 
     auto* rootPaintingLayer = enclosingSVGRootLayer();
+    if (is<RenderSVGHiddenContainer>(parent()->renderer()))
```

Comment:
Can you explain that?

Reply:
Yeah this is tricky. It fixes custom/pattern-userSpaceOnUse-userToBaseTransform.xhtml.
The outer svg's have a 1px border which is mistakenly taken into account (before that change) and offsets the pattern contents by 1x1, showing the red rectangle.

Reply:
I think it deserves a comment.